pkg/apis/serving/v1beta1/inference_service_defaults_test.go

```diff
@@ -897,7 +897,8 @@ func TestLocalModelAnnotation(t *testing.T) {
 					Predictor: PredictorSpec{
 						PyTorch: &TorchServeSpec{
 							PredictorExtensionSpec: PredictorExtensionSpec{
-								StorageURI: proto.String("gs://testbucket2/testmodel"),
+								// StorageURI has to match the one in LocalModelCache, extra suffix is not considered as a match
+								StorageURI: proto.String("gs://testbucket/testmodel/foo"),
```

Comment:
it would be nice to have a test that loads two similar models and test if the correct ones were loaded where it is needed.

Reply:
good point. Thank you!